### PR TITLE
POC: Add Client (WIP)

### PR DESCRIPTION
**Pull request by Mirsaid04**
@JohnnySenior

### CODE REVIEW

File: Tarteeb.Importer/Services/Clients/ClientService.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        private dynamic IsInvalid(string text) => new
+        {
+            Condition = String.IsNullOrWhiteSpace(text),
+            Message = "Text is required"
+        };
+
+        private dynamic IsInvalidEmail(string email) => new
+        {
+            Condition = !Regex.IsMatch(email, "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?"),
+            Message = "Invalid email"
```
Comment (ElbekDeveloper) [inline Tarteeb.Importer/Services/Clients/ClientService.cs]: Make message more informative

**Review by ElbekDeveloper — CHANGES_REQUESTED** [review]

File: Tarteeb.Importer/Models/Clients/Exceptions/ClientDependencyException.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿//=========================
+// Copyrite (c) Tarteeb LLC 
+// Powering True Leadership
+//=========================
+
+
```
Comment (ZafarUrakov) [inline Tarteeb.Importer/Models/Clients/Exceptions/ClientDependencyException.cs:6]: Remove extra space! @Mirsaid04 @JohnnySenior

File: Tarteeb.Importer/Models/Clients/Exceptions/ClientDependencyValidationException.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿//=========================
+// Copyrite (c) Tarteeb LLC 
+// Powering True Leadership
+//=========================
+
```
Comment (ZafarUrakov) [inline Tarteeb.Importer/Models/Clients/Exceptions/ClientDependencyValidationException.cs:5]: Remove extra space! @Mirsaid04 @JohnnySenior

File: Tarteeb.Importer/Models/Clients/Exceptions/ClientServiceException.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿//=========================
+// Copyrite (c) Tarteeb LLC 
+// Powering True Leadership
+//=========================
+
```
Comment (ZafarUrakov) [inline Tarteeb.Importer/Models/Clients/Exceptions/ClientServiceException.cs:5]: Remove extra space! @Mirsaid04 @JohnnySenior

File: Tarteeb.Importer/Models/Clients/Exceptions/FailedServiceClientException.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+﻿//=========================
+// Copyrite (c) Tarteeb LLC 
+// Powering True Leadership
+//=========================
+
```
Comment (ZafarUrakov) [inline Tarteeb.Importer/Models/Clients/Exceptions/FailedServiceClientException.cs:5]: Remove extra space! @Mirsaid04 @JohnnySenior

File: Tarteeb.Importer/Models/Clients/Exceptions/LockedClientException.cs [hunk 6]
```diff
@@ -0,0 +1,18 @@
+﻿//=========================
+// Copyrite (c) Tarteeb LLC 
+// Powering True Leadership
+//=========================
+
```
Comment (ZafarUrakov) [inline Tarteeb.Importer/Models/Clients/Exceptions/LockedClientException.cs:5]: Remove extra space! @Mirsaid04 @JohnnySenior

**Review by ZafarUrakov — CHANGES_REQUESTED** [review]
Correct the mistakes.

**Pull request merged**